client/components/TestRun/index.jsx

```diff
@@ -390,34 +391,53 @@ const TestRun = () => {
                 for (let i = 0; i < behaviors.length; i++) {
                     const behavior = behaviors[i];
                     if (behavior.checked) {
-                        if (i === 0)
-                            unexpectedBehaviors.push({
-                                id: 'EXCESSIVELY_VERBOSE'
-                            });
-                        if (i === 1)
-                            unexpectedBehaviors.push({
-                                id: 'UNEXPECTED_CURSOR_POSITION'
-                            });
-                        if (i === 2)
-                            unexpectedBehaviors.push({ id: 'SLUGGISH' });
-                        if (i === 3)
-                            unexpectedBehaviors.push({ id: 'AT_CRASHED' });
-                        if (i === 4)
-                            unexpectedBehaviors.push({ id: 'BROWSER_CRASHED' });
+                        if (i === 0) {
```

Comment:
REVIEWERS: this is the main concern here. Why does graphql expect a string while the TestRenderer expect an object with an id?